source/features/go-to-action-from-file.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import {getRepoURL, getRepoPath} from '../libs/utils';
+
+const isWorkflowFile = (): boolean => /\/\.github\/workflows\/.+\.ya?ml$/.test(getRepoPath()!);
+
+function init(): void {
+	const actionName = select.all('.blob-code-inner')
+		.find(line => line.textContent!.startsWith('name'))!
```

Comment:
This might be missing, so you’ll need `return false`

Reply:
With `if`?